packages/neuron-wallet/src/models/asset-account-info.ts

```diff
@@ -0,0 +1,85 @@
+import CellDep, { DepType } from "./chain/cell-dep"
+import Script, { ScriptHashType } from "./chain/script"
+import OutPoint from "./chain/out-point"
+import NetworksService from "services/networks"
+
+export interface ScriptCellInfo {
+  cellDep: CellDep
+  codeHash: string
+  hashType: ScriptHashType
+}
+
+export default class AssetAccountInfo {
+  private sudtInfo: ScriptCellInfo
+  private anyoneCanPayInfo: ScriptCellInfo
+  public sudtDeployHeight: bigint
+
+  private static MAINNET_GENESIS_BLOCK_HASH: string = '0x92b197aa1fba0f63633922c61c92375c9c074a93e85963554f5499fe1450d0e5'
+
+  public get infos(): {[name: string]: ScriptCellInfo} {
+    return {
+      sudt: this.sudtInfo,
+      anyoneCanPay: this.anyoneCanPayInfo
+    }
+  }
+
+  constructor(genesisBlockHash: string = NetworksService.getInstance().getCurrent().genesisHash) {
```

Comment:
It might be better to encapsulate these cell deps in a JSON file, which would be easier for maintenance and more flexible for testings.